components/automate-ui/src/app/pages/+compliance/compliance-reporting-sidebar/compliance-reporting-sidebar.component.spec.ts

```diff
@@ -117,15 +117,15 @@ describe('ComplianceReportingSidebarComponent', () => {
       return;
     }
     const firstCheckItem = permsFromLandingComp[0];
-    let [ path, verb ] = permsFromTemplate.split(',');
-    // If permsFromTemplate path is more than 30 characters, 
-    // verb will be undefined. We will be unable to test the verb 
+    const [ path, verb ] = permsFromTemplate.split(',');
+    // If permsFromTemplate path is more than 30 characters,
+    // verb will be undefined. We will be unable to test the verb
     // and need to mark firstCheckItem[1] as undefined to pass test.
     // AND If permsFromTemplate verb charaters is less than the
```

Comment:
s/charaters/characters/

Reply:
@msorens Could you clarify your comment?

Reply:
That is just vi shorthand to correct the word "charaters" to "characters".